components/infoColumns/index.tsx

```diff
@@ -98,57 +98,65 @@ const InfoColumns = () => {
           post-graduates, and professionals from backgrounds in academia,
           finance, politics, business, technology and more to deliver a
           high-caliber educational experience to our delegates.
+          <br />
+          <br />
+          This coming year, we welcome you - delegates, faculty, sponsors, and all others -
+          <em> in-person</em>, to the <strong> 18th Chicago International Model United Nations
+          conference</strong>. We hope you can make it!
         </Body>
       </CardContainer>
-      <CardWrapper flex="2" margins="1rem 0" pad="1rem 0 1rem 1rem">
+      <CardWrapper flex="2" margins="1rem 0" pad="1rem 0 6rem 1rem">
         <CardContainer
           bordered
           height="600px"
-          color={colors.fadedPrimaryBlue}
+          color={colors.primaryBlue}
           margins="1rem 1rem 0 0"
         >
-          <SubTitle size="2.5rem" margins="20px 0" self="center">
-            Join Us!
+          <SubTitle size="3.5rem" margins="10px 0" self="center" align="center" line={0.9} color={colors.ltGray}>
```

Comment:
You'll need to refactor these into the Title styled component so that you can add a breakpoint or else they will be much larger than other title on mobile. Usually weird to have a title down below be larger than the first prominent title in the section